index.js

```diff
@@ -10,7 +10,7 @@ module.exports = moduleId => {
 
 	const parentPath = parentModule(__filename);
 
-	const filePath = resolveFrom(path.dirname(parentPath), moduleId);
+	const filePath = resolveFrom(path.dirname(parentPath || moduleId), moduleId);
```

Comment:
Using `moduleId` is definitely no correct here. I would do:

```suggestion
	const cwd = parentPath ? path.dirname(parentPath) : __dirname;
	const filePath = resolveFrom(cwd, moduleId);
```

Reply:
Thanks for your review and your correction 

8ad1b3e27865cdfefba40ae3f5353cef7a5bd0c3 should fix the issue